doc/documentation_data_format.rst

```diff
@@ -15,6 +15,27 @@ systems biology, especially for the case of Ordinary Differential Equation
 (ODE) models.
 
 
+Scope
+-----
+
+The scope of PEtab is the full specification of parameter estimation problems
+in typical systems biology applications. In our experience, a typical setup of
+data-based modeling starts either with (i) the model of a biological system
+that is to be calibrated, or with (ii) experimental data that are to be
+integrated and analyzed using a computational model.
+Measurements are linked to the biological model by an observation and noise
+model. Often, measurements are taken after some perturbations have been
+applied, which are modeled as derivations from a generic model
+(Subfigure A below). Therefore, one goal was to specify such a setup in the
```

Comment:
This document is also auto-converted as part of the supplement, and in the pdf "below" may be wrong. is it possible to assign a figure index?

Reply:
Changed.